src/Broadway/EventStore/InMemoryEventStore.php

```diff
@@ -46,25 +48,35 @@ public function append($id, DomainEventStreamInterface $eventStream)
     {
         $id = (string) $id;
 
-        if (! isset($this->events[$id])) {
-            $this->events[$id] = [];
+        $aggregateEvents = [];
+        if (isset($this->events[$id])) {
+            $aggregateEvents = $this->events[$id];
         }
 
         foreach ($eventStream as $event) {
-            $playhead = $event->getPlayhead();
-            $this->assertPlayhead($this->events[$id], $playhead);
-
-            $this->events[$id][$playhead] = $event;
+            $aggregateEvents = $this->insertEvent($aggregateEvents, $event, $eventStream);
         }
+
+        $this->events[$id] = $aggregateEvents;
```

Comment:
Sorry I'm sure I wrote this down, think commits got messed up. Can you reference / explain the bug?